src/documents/tokenPrefsServerIntegrationConnection.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Token and Preferences Server Integration Connector
+layout: default
+category: API
+---
+
+## Overview
+
+**Component Name:** `gpii.firstDiscovery.panel.token.prefsServerIntegrationConnection`
+
+**File:** `panels.js`
+
+This connector sends a http request to the first discovery server to save user selected preferences when the token panel becomes visible.
```

Comment:
1) This sentence should read "an http request" instead of "a http request".
2) I think everywhere else in the document, we refer to it as a "preferences server" and not a "first discovery server" – unless this is something different?